.kitchen.yml

```diff
@@ -247,3 +247,10 @@ suites:
       systems:
         - name: simple_autopilot_public
           backend: local
+  - name: "autopilot_private_firewalls"
```

Comment:
The Kitchen test method is deprecated for these repos, however I see you already included the current cft style `test/integration/autopilot_private_firewalls/autopilot_private_firewalls_test.go` test file.  Instead of adding to Kitchen you should be able to use CFT test by adding the required steps to `build/int.cloudbuild.yaml`

Here is an example for safer-cluster, yours will probably be called TestAutopilotPrivateFirewalls and could be placed right after the safer-cluster steps in `build/int.cloudbuild.yaml`

```
- id: apply autopilot_private_firewalls
  waitFor:
    - init create-all
  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
  args: ['/bin/bash', '-c', 'cft test run TestAutopilotPrivateFirewalls --stage apply --verbose']
- id: verify autopilot_private_firewalls
  waitFor:
    - apply autopilot_private_firewalls
  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
  args: ['/bin/bash', '-c', 'cft test run TestAutopilotPrivateFirewalls --stage verify --verbose']
- id: destroy autopilot_private_firewalls
  waitFor:
    - verify autopilot_private_firewalls
  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
  args: ['/bin/bash', '-c', 'cft test run TestAutopilotPrivateFirewalls --stage destroy --verbose']
  ```

Reply:
@apeabody Thankyou for feedback, have pushed changes. 
- removed the kitchen test config for autopilot-private-firewalls
- added cft test config. 

Local `make docker_test_lint` was happy with the cloudbuild yaml so hopefully all good. Can we take it for a spin?